tests/SpecificUnitTest.txt

```diff
@@ -1,6 +1,7 @@
 testTitle=UnitTests
 startDelay=0
 useDB=false
+runSetup=false
```

Comment:
Couldn't there be other unit test cases that relies on `runSetup` being completed?

Reply:
A unit test should not require setting up the entire DB. Otherwise, it should be categorized as cluster test.

Reply:
That's a reasonable expectation. CI run should tell us if it holds.